plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryWithDifferentProjectIdConnectorSmokeTest.java

```diff
@@ -0,0 +1,115 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.bigquery;
+
+import com.google.common.collect.ImmutableMap;
+import io.trino.plugin.bigquery.BigQueryQueryRunner.BigQuerySqlExecutor;
+import io.trino.testing.BaseConnectorSmokeTest;
+import io.trino.testing.QueryRunner;
+import io.trino.testing.TestingConnectorBehavior;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+
+import java.util.Map;
+
+import static io.trino.testing.TestingNames.randomNameSuffix;
+import static java.util.Objects.requireNonNull;
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class TestBigQueryWithDifferentProjectIdConnectorSmokeTest
+        extends BaseConnectorSmokeTest
+{
+    private static final String ALTERNATE_PROJECT_CATALOG = "bigquery";
+    private static final String SERVICE_ACCOUNT_CATALOG = "service_account_bigquery";
+
+    protected BigQuerySqlExecutor bigQuerySqlExecutor;
+    protected String alternateProjectId;
+
+    @BeforeClass(alwaysRun = true)
+    public void initBigQueryExecutor()
+    {
+        this.bigQuerySqlExecutor = new BigQuerySqlExecutor();
+    }
+
+    @Override
+    protected QueryRunner createQueryRunner()
+            throws Exception
+    {
+        this.alternateProjectId = requireNonNull(System.getProperty("testing.alternate-bq-project-id"), "testing.alternate-bq-project-id system property not set");
+
+        QueryRunner queryRunner = BigQueryQueryRunner.createQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of("bigquery.project-id", alternateProjectId),
+                REQUIRED_TPCH_TABLES);
+        queryRunner.createCatalog(SERVICE_ACCOUNT_CATALOG, "bigquery", Map.of());
+        return queryRunner;
+    }
+
+    @SuppressWarnings("DuplicateBranchesInSwitch")
+    @Override
+    protected boolean hasBehavior(TestingConnectorBehavior connectorBehavior)
+    {
+        switch (connectorBehavior) {
+            case SUPPORTS_RENAME_SCHEMA:
+                return false;
+
+            case SUPPORTS_RENAME_TABLE:
+                return false;
+
+            default:
+                return super.hasBehavior(connectorBehavior);
+        }
+    }
+
+    @Test
+    public void testCreateSchemasInDifferentProjectIdCatalog()
+    {
+        // This test case would fail without the bug fix https://github.com/trinodb/trino/issues/14083
+        // It would create a schema in the wrong project, not the one defined in the catalog properties
+        String serviceAccountSchema = "service_account_schema" + randomNameSuffix();
+        String projectIdSchema = "project_id_schema" + randomNameSuffix();
+        try {
+            assertThat(getQueryRunner().execute("SHOW CATALOGS").getOnlyColumnAsSet())
```

Comment:
nit: for consistency with other classes (I know this is pre-existing style in BigQuery connector tests though)

```suggestion
            assertThat(computeActual("SHOW CATALOGS").getOnlyColumnAsSet())
```